client/src/commands/instantiateCommand.ts

```diff
@@ -73,6 +74,10 @@ export async function instantiateSmartContract(channelTreeItem?: ChannelTreeItem
             const fabricClientConnection: IFabricConnection = FabricConnectionManager.instance().getConnection();
             await fabricClientConnection.instantiateChaincode(data.chaincode, data.version, channelName, fcn, args);
 
+            const outputAdapter: VSCodeOutputAdapter = VSCodeOutputAdapter.instance();
+            const message: string = `\nInstantiated with command: '` + fcn + `' and arguments: '` + args + `'`;
```

Comment:
i think this should go before the `instantiateChaincode` function as even if it fails you still might want to see what you did